metadata.json

```diff
@@ -67,7 +67,10 @@
       ]
     },
     {
-      "operatingsystem": "Gentoo"
+      "operatingsystem": "Gentoo",
+      "operatingsystemrelease": [
+        "4"
```

Comment:
This shouldn't be needed and isn't right, but without it, the facter 3.11 and 3.14 facts aren't found...